charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -39,6 +39,8 @@ annotations:
       url: https://github.com/prometheus-operator/kube-prometheus
 
 dependencies:
+  - name: crds
+    version: "0.0.0"
```

Comment:
I would add a flag to enable/disable them the same way it's done for ksm, node exporter and grafana